appinventor/appengine/src/com/google/appinventor/client/components/dialog.css

```diff
@@ -0,0 +1,30 @@
+@external Caption;
+
+.dialog {
+  border-radius: 10px;
+  z-index: 1000;
+  background-color: #FFF;
+  width: 400px;
+  box-shadow: 4px 4px 20px 7px rgba(0, 0, 0, 0.1);
+  overflow: hidden !important;
+}
+
+.dialog table {
+  width: 100%;
+}
+
+.dialog .Caption {
+  padding: 10px 20px;
+  text-align: center;
+  border-bottom: 1px solid #DDD;
+  margin-bottom: 10px;
+  font-size: 18px;
+  font-family: 'Poppins';
+  font-weight: 500;
+}
+
+@if (com.google.appinventor.client.Ode.isMobile()) {
```

Comment:
As previously noted, we may want to use CSS media queries based on screen size to decide what to do with dialog rather than explicitly testing for mobile.

Reply:
This is a remnant of the prototype code that should have been removed. I've removed it. We expect to use this prototype code to build out a mobile view, but the prototype's mostly explored these areas and wasn't complete even for the project explorer.